libs/styles/rounded-corner.css

```
@@ -0,0 +1,29 @@
+:root {
+  /* border-radius sizes */
+  --s-rounded-corners: 4px;
+  --m-rounded-corners: 8px;
+  --l-rounded-corners: 16px;
+  --full-rounded-corners: 50%;
+}
+
+.s-rounded-corners,
+.s-rounded-corners > .background img,
+.s-image-rounded-corners .foreground .image picture img {
+  border-radius: var(--s-rounded-corners);
+}
+
+.m-rounded-corners,
```

```suggestion
.rounded-corners,
.m-rounded-corners,
```
We should set defaults for no size passed `rounded-corners` && `rounded-corners-image`

---

done